src/test/java/giis/eshopcontainers/e2e/functional/utils/Shopping.java

```diff
@@ -0,0 +1,89 @@
+package giis.eshopcontainers.e2e.functional.utils;
+
+import giis.eshopcontainers.e2e.functional.common.ElementNotFoundException;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Class with various methods facilitating the catalog-shopping actions
+ */
+public class Shopping {
```

Comment:
Package utils contains general purpose support classes, like navigation, clicking, etc. However this class is domain specific, more precisely, test specific (about shopping cart). Methods here should be part of the test that is using them (at least, in view of the tests that we have now)